### PR TITLE
feat(v2/avatar): update theme for Avatar and AvatarMenu to fit design

### PR DESCRIPTION
**Pull request by karrui**
Previous Avatar implementation did not 100% follow design. This PR updates the theme for Avatar and simplifies AvatarMenu props to be more general.

As a result, we no longer need a custom `Avatar` component and can use ChakraUI's default component in our `AvatarMenu` template component.

This will also unblock #4129.

### CODE REVIEW

**Review by tshuli — APPROVED** [review]
lgtm

File: frontend/src/theme/components/Avatar.ts [hunk 1]
```diff
@@ -1,24 +1,61 @@
...
+    },
+  }
+}
+
+const getBadgePlacement = (size: ThemingProps['size']): SystemStyleObject => {
+  switch (size) {
+    // Update width and height calculation in the future if needed.
+    default:
+      return {
+        botton: '1px',
```
Comment (timotheeg) [inline frontend/src/theme/components/Avatar.ts:27]:
```suggestion
        bottom: '1px',
```

/personalPreference: at xs size, it looks better with bottom 0 than bottom 1px

**Review by timotheeg — APPROVED** [review]
LGTM! One minor fix comment.

**Comment by timotheeg** [comment]
LOL, you too fast for me @karrui 😅😂!

File: frontend/src/theme/components/Avatar.ts (hunk 1 above)
Comment (karrui) [reply frontend/src/theme/components/Avatar.ts:27]: NOOOOO I MERGED

Comment (timotheeg) [reply frontend/src/theme/components/Avatar.ts:27]: Sorry! It wasn't merged when I opened it to take a look 😅

**Pull request merged**